### ISSUE TITLE
Refactor Team & BioinformaticsTeam

### ISSUE THREAD

**Issue by joncison** [IFB-ElixirFr/ifbcat#124]
``unitId`` and ``address`` fields should be moved from ``bioinformaticsTeam`` model to ``team``

 @jvanheld are there any other [fields](https://docs.google.com/spreadsheets/d/1tMzwkZINvFTj5mUOCBA8MrR8kxjrD6GRfyqH_mT1pi4/edit#gid=346104586) that should be moved ? 

cc @hmenager

**Comment by guillaumecharbonnier** [comment]
From what I think I could move these to team:

* orgid "Organisation ID (GRID or ROR ID) of the team."
* unitId "Unit ID (unique identifier of research or service unit) that the ~~Bioinformatics~~ Team belongs to."
* fields "A broad field that the ~~bioinformatics~~ team serves."
* keywords "A keyword ~~(beyond EDAM ontology scope)~~ describing the ~~bioinformatics~~ team."
* affiliatedWith "Organisation(s) to which the ~~bioinformatics~~ team is affiliated."
* communities "Communities in which the ~~bioinformatics~~ team is involved.",
* projects "Project(s) that the ~~bioinformatics~~ team is involved with, supports or hosts.",
* fundedBy "Organisation(s) that funds the ~~bioinformatics~~ team.",
* publications "Publication(s) that describe the team.",
* certification "Certification (e.g. ISO) of the ~~bioinformatics~~ team.",

And keep these to bioinformaticsTeam:

* topics "URIs of EDAM Topic terms describing the bioinformatics team."
* ifbMembership "Type of membership the bioinformatics team has to IFB."
* platforms "ELIXIR Platform(s) in which the bioinformatics team is involved.",

@jvanheld Could you confirm so I can do the changes and update the import in consequence?

**Comment by guillaumecharbonnier** [comment]
I have found this doc confirming my thinking with a few more elements:
https://docs.google.com/spreadsheets/d/1l9mj1UkJt6QCmN2gLZojkUmPflezZRwUYNlJ3aTFJW8/edit#gid=305603803
Please confirm this is the most up-to-date target I should aim for.

**Comment by jvanheld** [comment]
Hi Guillaume

We discussed yesterday with Nicolas and Suzanne about the way to display a team on the front-end, and we explored the different pieces of information available. I confirm that as far as I know the  spreadhseet you pointed is the most recent. We worked together on it for a few hours. I am not sure it is frozen yet, I think it should still evolve but I think it is the best starting point. 

Additional elements might be found in two sources: 

1. The data imported in the catalogue, already available via the REST interface : 
https://catalogue.france-bioinformatique.fr/api/team/

2. The data model developed by Jon, with Hervé's help and mine.

3. The pages "Plateformes" of the previous IFB Drupal site.

Cheers

Jacques
